go-controller/pkg/util/util.go

```diff
@@ -27,7 +27,6 @@ const (
 	// access to local service
 	LocalNetworkName = "locnet"
 
-	// FIXME DUAL-STACK
```

Comment:
what needed to be fixed here :)

Reply:
it still needs to be fixed, but the code that actually uses those constants is in pkg/ovn/master.go, and also has its own "`// FIXME DUAL-STACK SUPPORT`" comment, and I don't think the constants themselves are going to need to change so we don't need a comment here